docs/dev_guide/contents/extending_fido.rst

```diff
@@ -289,6 +289,18 @@ An Example of ``register_values()``
 
         return adict
 
+Registering custom attrs in the ``attrs`` namespace
+---------------------------------------------------
+When you have custom attrs defined in a separate attrs module, you can add them to the namespace using the :meth:`~sunpy.net.BaseClient._attrs_module` class method.
+The method returns a tuple having the name of the target namespace and the module to be imported in that namespace.
```

Comment:
```suggestion
The method returns a tuple of length 2, where the first element is the name you want to add to the main attrs namespace. The second is module where the custom attrs exist, so that can be imported.
```